lua/statuscol.lua

```diff
@@ -185,8 +185,11 @@ local function place_signs(win, signs, ext)
     local s = ext and signs[i][4] or signs[i]
     local name = ext and s.sign_text and s.sign_text..s.sign_hl_group or s.name
     if ext and not name then goto nextsign end
-    if not sign_cache[name] then update_sign_defined(win, ext and signs) end
     local sign = sign_cache[name]
+    if not sign then
+      update_sign_defined(win, ext and signs)
+      goto nextsign
```

Comment:
Without this change the next line with `sign.segment` will try to index a nil value

Reply:
Are you using neovim 0.10 (nightly) on this new device? There is a 0.10 branch for this plugin which should be used with neovim nightly.

`sign` shouldn't be `nil` after updating `sign_cache`.

Reply:
That's true!
![image](https://github.com/luukvbaal/statuscol.nvim/assets/33424304/7fea063f-7f5b-4273-9848-89d935d297d3)

Thank you! I didn't know there was a branch for nightly version. I will close the PR.

Reply:
Thanks 👍